packages/trpc/server/routers/viewer/teams/create.handler.ts

```diff
@@ -17,10 +18,15 @@ type CreateOptions = {
 
 export const createHandler = async ({ ctx, input }: CreateOptions) => {
   const { slug, name, logo } = input;
-
+  const currentOrgId = ctx.user.organization?.id;
   const slugCollisions = await prisma.team.findFirst({
     where: {
       slug: slug,
+      parentId: currentOrgId
+        ? {
+            equals: currentOrgId,
+          }
+        : null,
```

Comment:
Allows compatability for create teams and also child teams for an org from this function. 

Prevents a lot of changes on the FE to conditionally call this.